packages/validator/src/services/indices.ts

```diff
@@ -108,6 +108,13 @@ export class IndicesService {
         this.pubkey2index.set(pubkeyHex, validatorState.index);
         this.index2pubkey.set(validatorState.index, pubkeyHex);
         newIndices.push(validatorState.index);
+      } else {
+        this.logger.info(
+          `Validator:
```

Comment:
We try to format logs like:
`logger[method](message, obj)`
Where we think about each of the three params:
- method - very conscious about the info / error logs we add as they can confuse or scare users
- message - Informative but not overly verbose
- obj - a single-level object (check its type) with useful and relevant metadata

so this one could look like:
`logger.info("Discovered validator", {status: validatorState.status ?? "unknown, index: this.pubkey2index.get(pubkeyHex), pubkey: pubkeyHex})`